src/components/Signup.jsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState } from "react";
+import SalonPicSignup1 from "../images/signup/salonpicsignuppage1.jpeg";
+import SalonPicSignup2 from "../images/signup/salonpicsignuppage2.jpeg";
+import Line from "./Line";
+import axios from "axios";
+import { ToastContainer, toast, Zoom } from "react-toastify";
+import 'react-toastify/dist/ReactToastify.css';
+import * as EmailValidator from "email-validator"
+
+const Signup = () => {
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [name, setName] = useState("");
+    const [phone, setPhone] = useState("");
+    const [budget, setBudget] = useState("");
+    const [gender, setGender] = useState("");
+    const [city, setCity] = useState("");
```

Comment:
can we wrap this all in an object.
objec.email
objec.passwrod
    const [user, setUser] = useState({});